universal-application-tool-0.0.1/conf/routes

```diff
@@ -21,6 +21,13 @@ GET     /admin/programs/:id/edit    controllers.admin.AdminProgramController.edi
 POST    /admin/programs/:id         controllers.admin.AdminProgramController.update(request: Request, id: Long)
 POST    /admin/programs             controllers.admin.AdminProgramController.create(request: Request)
 
+# A controller for a page for an admin to view, edit, and create questions
+GET     /admin/questions             controllers.admin.QuestionController.list(renderAs: String ?= "table")
+GET     /admin/questions/new         controllers.admin.QuestionController.create(request: Request)
+GET     /admin/questions/edit/:path  controllers.admin.QuestionController.edit(request: Request, path: String)
+POST    /admin/questions/update      controllers.admin.QuestionController.update(request: Request)
+POST    /admin/questions/write       controllers.admin.QuestionController.write(request: Request)
```

Comment:
suggestion (blocking): make sure these align with https://github.com/seattle-uat/universal-application-tool#routing-and-controller-methods

Reply:
Done.